library/eggdrop

```diff
@@ -1,10 +1,14 @@
 Maintainers: Geo Van O <geo@eggheads.org> (@vanosg)
 GitRepo: https://github.com/eggheads/eggdrop-docker.git
 
-Tags: 1.8, 1.8.0, develop
-GitCommit: 49321b7d9b37d7a177a9a1b9c5f14eb13df5e204
+Tags: develop
```

Comment:
Naw, that's fair -- twice a month shouldn't be a big deal at all. :+1:  Just checking!  Thanks! :heart: